doc/libpmem2/pmem2_deep_sync_fn.3.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEM2_GET_FLUSH_FN, 3)
+collection: libpmem2
+header: PMDK
+date: pmem2 API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmem2_deep_sync_fn.3 -- man page for pmem2_deep_sync_fn)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmem2_deep_sync_fn**() - high reliability persistent memory synchronization
```

Comment:
why _fn?
please add this man to doc/Makefile,  doc/libpmem2/.gitignore and _data/releases_* on gh-pages

Reply:
typo.

I modified these files, but I forgot to `git add` them. Done.